### PR TITLE
Outline for Review Paper

### PR DESCRIPTION
**Pull request by danich1**
Uploading an outline for the review paper. Let me know your thoughts for this outline.

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+1. Talk about knowledge graphs
+  1. Define what they are
+  2. Provide a diagram to orient the reader
+2. Mention how these graphs are constructed 
+3. Mention some applications these graphs can be used for
+	1. Biomedical setting
+	2. Social Media 
+	3. Question and answering systems 
+	4. more if it comes to mind
+4. Conclude section by mention that this review dives deeper into approaches used for construction and applying these graphs
```
Comment (cgreene) [inline content/02.introduction.md]: Might be good to note towards the conclusion once more that we're focusing on biomedical applications

File: content/03.text-mining.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## Relationship Extraction
+
+1. Set up the context for relationship extraction
+	1. Define the problem
+	2. Talk about the importance of the problem (filling knowledge bases -> point researchers to relevant papers)
+2. Give overview towards taxonomy of approaches (hand written rules, unsupervised machine learning, supervised machine learning etc.)
+
+### Text Mining
+
+#### Hand written Natural Language Processing Techniques
```
Comment (cgreene) [inline content/03.text-mining.md]: Are these relevant to biomedical knowledge graph extraction?

Comment (cgreene) [reply content/03.text-mining.md]: Or - wait - do you mean of hand written text, or do you mean like human-written rules? If so, I might define these as:
> Rule-based NLP

File: content/03.text-mining.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+## Relationship Extraction
+
+1. Set up the context for relationship extraction
+	1. Define the problem
+	2. Talk about the importance of the problem (filling knowledge bases -> point researchers to relevant papers)
+2. Give overview towards taxonomy of approaches (hand written rules, unsupervised machine learning, supervised machine learning etc.)
+
+### Text Mining
```
Comment (cgreene) [inline content/03.text-mining.md]:
```suggestion
### Text Mining for Relationship Extraction
```

File: content/03.text-mining.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+## Relationship Extraction
```
Comment (cgreene) [inline content/03.text-mining.md]:
```suggestion
## Building Biomedical Knowledge Graphs
```

File: content/03.text-mining.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Set up the context for relationship extraction
+	1. Define the problem
+	2. Talk about the importance of the problem (filling knowledge bases -> point researchers to relevant papers)
+2. Give overview towards taxonomy of approaches (hand written rules, unsupervised machine learning, supervised machine learning etc.)
```
Comment (cgreene) [inline content/03.text-mining.md:6]: I think you will want another subsection on things like database construction & manual curation (non-text mining).

Are there approaches to building biomedical knowledge graphs beyond text mining or the database-based strategy?

File: content/04.network_applications.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+## Knowledge Graph Applications
```
Comment (cgreene) [inline content/04.network_applications.md]:
```suggestion
## Applying Knowledge Graphs to Biomedical Challenges
```

File: content/04.network_applications.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+1. Mention that these graphs can be used for discovery
+2. Mention representation learning (aka representing a graph as dense vectors for nodes and/or edges)
+3. 
+
```
Comment (cgreene) [inline content/04.network_applications.md:6]: You have both techniques and applications here. What do you think about talking about the same literature but from two perspectives. You'd have a subsection at the `###` level called "Unifying Techniques" and another called "Unifying Applications". This would help readers approach the literature from either angle. In both of these subsections, you'd approach the same papers.

**Review by cgreene — COMMENTED** [review]
I really like where this is going! I had a few thoughts on organization that might make the review easier to author and also easier to read. See what you think! If you want to do a bit of reorganization towards this structure, update the outline and then ping me again for another look through. Thanks!

File: content/03.text-mining.md (hunk 2 above)
Comment (danich1) [reply content/03.text-mining.md]: Meant human-written rules. Have made a change according to the above comment.

**Review by cgreene — APPROVED** [review]
